react/EditorProvider.tsx

```diff
@@ -26,7 +26,9 @@ interface State {
   viewport: Viewport
 }
 
-let iframeMessages: Record<string, string>
+let iframeMessages: Record<string, string> | undefined
```

Comment:
:pick: Perhaps we should keep using `<var>?: ...` instead of ` <var>: ... | undefined` for consistency.